qiskit/algorithms/state_fidelities/compute_uncompute.py

```diff
@@ -49,14 +49,29 @@ class ComputeUncompute(BaseStateFidelity):
 
     """
 
-    def __init__(self, sampler: BaseSampler, options: Options | None = None) -> None:
-        """
+    def __init__(
+        self,
+        sampler: BaseSampler,
+        options: Options | None = None,
+        local: bool = False,
+    ) -> None:
+        r"""
         Args:
             sampler: Sampler primitive instance.
             options: Primitive backend runtime options used for circuit execution.
                 The order of priority is: options in ``run`` method > fidelity's
                 default options > primitive's default setting.
                 Higher priority setting overrides lower priority setting.
+            local: If set to ``True``, the fidelity is averaged over
+                single-qubit projectors
+                .. math::
+
+                    \hat{O} = \frac{1}{N}\sum_{i=1}^N|0_i\rangle\langle 0_i|,
+
```

Comment:
This is not formatted correctly:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/5978796/216074448-5a2bd034-68ba-453d-8a9a-442bf811f9dc.png">


I think you have to add an additional linebreak before the `.. math` -- but if that doesn't work then you could also put this info in the class docstring 🙂

Reply:
Thanks, adding the extra line break worked 👍